src/sentry/integrations/jira/integration.py

```diff
@@ -145,6 +146,13 @@ def get_organization_config(self):
             configuration[0]['disabledReason'] = _(
                 'Unable to communicate with the Jira instance. You may need to reinstall the addon.')
 
+        organization = Organization.objects.get(id=self.organization_id)
+        has_issue_sync = features.has('organizations:integration:issue_sync',
+                                      organization)
+        if not has_issue_sync:
+            for field in configuration:
+                field['disabled'] = True
```

Comment:
You can add `disabledReason` too, which might be a quick workaround

Reply:
Though really we should do some hookstore stuff, since that won't make sense for onprem

Reply:
ok i'll do somethign generic like "Your organization doesn't have this feature" for now and we can get soemthign w hookstore up in the next couple days